larq_zoo/literature/binary_alex_net.py

```diff
@@ -91,9 +92,9 @@ def build(self) -> tf.keras.models.Model:
             if self.include_top:
                 weights_path = utils.download_pretrained_model(
                     model="binary_alexnet",
-                    version="v0.2.0",
+                    version="v0.3.0",
                     file="binary_alexnet_weights.h5",
```

Comment:
These weights have been manually reshaped from previous model so that Keras can correctly reload them.